### PR TITLE
Allow Quick Access Links to be accessed with middle string

### PR DESCRIPTION
**Pull request by taooceros**
Use `Contains` instead of `StartWith` to allows search with middle part of name, also score bump to 100 since it is not a fuzzy, but a full matched.

Or using Fuzzy would be better?

### CODE REVIEW

**Comment by jjw24** [comment]
Given rest of search in Explorer can't use fuzzy, probably just stick with contains so not too confuse users?

Instead of looping through the matched results again and update score, what about adding an optional score parameter to the create file and create folder methods and assign the score that way?

**Comment by taooceros** [comment]
> Given rest of search in Explorer can't use fuzzy, probably just stick with contains so not too confuse users?

I guess the rest of the search in Explorer still uses fuzzy to calculate the score.

> Instead of looping through the matched results again and update score, what about adding an optional score parameter to the create file and create folder methods and assign the score that way?

Yeah that seems great,.

**Comment by jjw24** [comment]
> I guess the rest of the search in Explorer still uses fuzzy to calculate the score.

Just to calculate score or also uses fuzzy? I'm afk so can't take a look...

**Comment by taooceros** [comment]
> > I guess the rest of the search in Explorer still uses fuzzy to calculate the score.
> 
> Just to calculate score or also uses fuzzy? I'm afk so can't take a look...

calculate the score as I remember previously. Let met take a look

**Comment by taooceros** [comment]
Update, it utilizes fuzzy to create highlighed data......

**Comment by jjw24** [comment]
Ok so actual search is not using fuzzy yeah

**Comment by jjw24** [comment]
Version bump also forgot to mention

File: Plugins/Flow.Launcher.Plugin.Explorer/Search/WindowsIndex/IndexSearch.cs [hunk 1]
```diff
@@ -53,11 +53,11 @@ internal async static Task<List<Result>> ExecuteWindowsIndexSearchAsync(string i
                                                                     dataReaderResults.GetString(0),
                                                                     path,
                                                                     path,
-                                                                    query, true, true));
+                                                                    query, 0, true, true));
                             }
                             else
                             {
-                                fileResults.Add(ResultManager.CreateFileResult(path, query, true, true));
+                                fileResults.Add(ResultManager.CreateFileResult(path, query, 0, true, true));
```
Comment (jjw24) [inline Plugins/Flow.Launcher.Plugin.Explorer/Search/WindowsIndex/IndexSearch.cs:60]: Why are we setting the score to 0 in several of the changes, have they always been 0?

Comment (taooceros) [reply Plugins/Flow.Launcher.Plugin.Explorer/Search/WindowsIndex/IndexSearch.cs:60]: Yeah, we don't set the score before, which is default 0.

**Review by jjw24 — APPROVED** [review]

**Comment by taooceros** [comment]
Thanks, haven't got time to finish the version bump.

**Pull request merged**